backend/api/Controllers/ScheduledMissionController.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Api.Controllers.Models;
+using Api.Models;
+using Api.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Api.Controllers;
+
+[ApiController]
+[Route("scheduledMission")]
+public class ScheduledMissionController : ControllerBase
+{
+    private readonly ScheduledMissionService _scheduledMissionService;
+    private readonly RobotService _robotService;
+
+    public ScheduledMissionController(ScheduledMissionService scheduledMissionService, RobotService robotService)
+    {
+        _scheduledMissionService = scheduledMissionService;
+        _robotService = robotService;
+    }
+
+    /// <summary>
+    /// Gets a list of the scheduled missions in the database.
+    /// </summary>
+    /// <remarks>
+    /// <para> This query gets all scheduled missions (paginated) </para>
+    /// </remarks>
+    [HttpGet]
+    [ProducesResponseType(typeof(IList<ScheduledMission>), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<ActionResult<IList<ScheduledMission>>> GetScheduledMissions()
+    {
+        var scheduledMissions = await _scheduledMissionService.ReadAll();
+        return Ok(scheduledMissions);
+    }
+
+    /// <summary>
+    /// Lookup scheduled mission with specified id
+    /// </summary>
+    /// <remarks>
+    /// <para> This query gets the scheduled mission with the specified id </para>
+    /// </remarks>
+    [HttpGet]
+    [Route("{id}")]
+    [ProducesResponseType(typeof(ScheduledMission), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<ActionResult<ScheduledMission>> GetScheduledMissionById([FromRoute] string id)
+    {
+        var scheduledMission = await _scheduledMissionService.Read(id);
+        if (scheduledMission == null)
```

Comment:
```suggestion
        if (scheduledMission is null)
```
C# best practices is to use "is" as the null comparison because it avoids the risk of overloaded == operators and increases readability.